conversion/conversion.go

```diff
@@ -115,7 +116,7 @@ func DataConv(driver string, ioHelper *IOStreams, client *sp.Client, conv *inter
 		return dataFromSQL(driver, config, client, conv)
 	case PGDUMP, MYSQLDUMP:
 		if conv.SpSchema.CheckInterleaved() {
-			return nil, fmt.Errorf("HarbourBridge does not currently support data conversion from dump files\nif the schema contains interleaved tables. Suggest using direct access to source database\ni.e. using drivers postgres and mysql.")
+			return nil, fmt.Errorf("harbourBridge does not currently support data conversion from dump files\nif the schema contains interleaved tables. Suggest using direct access to source database\ni.e. using drivers postgres and mysql")
```

Comment:
Should proper nouns like HarbourBridge also be changed to lowercase?

Reply:
I think ideally no, but the golinter was complaining about this